source/administration/command-line-tools.rst

```diff
@@ -437,7 +438,28 @@ mattermost export actiance
     .. code-block:: none
 
           --exportFrom string     Unix timestamp (seconds since epoch, UTC) to export data from.
-	 
+
+mattermost export bulk
+~~~~~~~~~~~~~~~~~~~~~
+
+  Description
+    Export data to a file compatible with the Mattermost Bulk Import format.
+
+  Format
+    .. code-block:: none
+
+      mattermost export bulk
+
+  Example
+    .. code-block:: none
+
+      sudo ./mattermost export bulk --all-teams file.json
+
+  Options
+    .. code-block:: none
+
+          --all-teams bool     Export all teams from the server.
```

Comment:
What is exported if this option is not used?

Reply:
@grundleborg Can you also confirm what happens if this option is not used.  It looks as though it may be required?

Reply:
Confirmed with GG and updated docs